pl_bolts/models/self_supervised/moco/moco2_module.py

```diff
@@ -121,6 +121,12 @@ def __init__(
 
         self.register_buffer("queue_ptr", torch.zeros(1, dtype=torch.long))
 
+        # create the validation queue
+        self.register_buffer("val_queue", torch.randn(emb_dim, num_negatives))
+        self.queue = nn.functional.normalize(self.val_queue, dim=0)
```

Comment:
As far as I understood, there is a small bug here, we should assign normalized `self.val_queue` to itself, but not to `self.queue`. I will fix it in the following commits